.kokoro/tests/run_tests.sh

```diff
@@ -166,7 +166,7 @@ for file in **/pom.xml; do
     fi
 
     # Use maven to execute the tests for the project.
-    mvn -q --batch-mode --fail-at-end clean verify \
+    mvn -q --batch-mode --fail-at-end clean verify -P lint \
```

Comment:
nit: Does this need to be at the beginning?
```suggestion
    mvn -P lint -q --batch-mode --fail-at-end clean verify \
```